modeltranslation_wagtail/translator.py

```diff
@@ -13,12 +12,13 @@
     pass
 
 
-class TranslationOptions(TranslationOptionsOrig):
+class TranslationOptionsWagtail(TranslationOptions):
```

Comment:
I don't agree with this, since it breaks my own stuff :) Already released it with the former name. If you want, you can always use `from  modeltranslation_wagtail.translator import TranslationOptions as TranslationOptionsWagtail`

Reply:
yes, fair enough :)